components/LeftNavbar.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="h-full border-r border-secondary-bg/40">
+    <div class="h-full border-r border-secondary-bg/40 hover:shadow-inner hover:shadow-secondary-bg/50 transition-shadow">
```

Comment:
Nit: this feels like a bit overkill. What's the benefit to having a hover on this div instead of just its child elements?

Reply:
two reasons:
1) having it on the parent element ensures the whole container transitioned. What often happens is someone will add a margin to the child element, or more elements will be added to the left navbar and it can be forgotten
2) when possible, it's better css performance to use effects on a singular container vs multiple child components.....when possible